peter-lang-verlag.csl

```diff
@@ -0,0 +1,161 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="note" version="1.0" page-range-format="expanded">
+  <info>
+    <title>Peter Lang Verlag</title>
+    <title-short>Peter Lang</title-short>
+    <id>http://www.zotero.org/styles/peter-lang-verlag</id>
+    <link href="http://www.zotero.org/styles/peter-lang-verlag" rel="self"/>
+    <author>
```

Comment:
if possible, add a rel="template" link for the template you based your style on and also a rel="documentation" link with a link to the documentation.